src/Common/src/Internal/Runtime/ModuleHeaders.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+
+namespace Internal.Runtime
+{
+    //
+    // ModuleHeaderSection IDs are used by the runtime to look up specific global data sections
+    // from each module linked into the final binary. New sections should be added at the bottom
+    // of the enum and deprecated sections should not be removed to preserve ID stability.
+    //
+    enum ModuleHeaderSection
+    {
```

Comment:
This should be eventually reconciled with ReadyToRunSectionType from https://github.com/dotnet/coreclr/blob/master/src/inc/readytorun.h. Could you please  add comment about it?

Looking at what is there today - we should start this set of IDs at 200.